cert-manager/src/types/certificate.ts

```diff
@@ -0,0 +1,58 @@
+import { KubeObject } from '@kinvolk/headlamp-plugin/lib/k8s/cluster';
```

Comment:
/types/ folder name is a bit misleading. types usually means only typescript type definitions, here we define classes. Maybe /model/ would be better?

Reply:
changed it to resource as in k8s follows that naming convention.